superset-frontend/src/dashboard/actions/dashboardInfo.ts

```diff
@@ -114,28 +114,28 @@ export const setChartConfiguration =
       dispatch({
         type: SET_CHART_CONFIG_COMPLETE,
         chartConfiguration,
+        globalChartConfiguration,
       });
     } catch (err) {
-      dispatch({ type: SET_CHART_CONFIG_FAIL, chartConfiguration });
+      dispatch({
+        type: SET_CHART_CONFIG_FAIL,
```

Comment:
To align these with the new function name that changes `set` to `save`, could we rename these to `SAVE_CHART_CONFIG_*`?